test/mpi/part/Makefile.am

```diff
@@ -0,0 +1,39 @@
+##
+## Copyright (C) by Argonne National Laboratory
+##     See COPYRIGHT in top-level directory
+##
+
+include $(top_srcdir)/Makefile_single.mtest
+
+EXTRA_DIST = testlist
+
+## for all programs that are just built from the single corresponding source
+## file, we don't need per-target _SOURCES rules, automake will infer them
+## correctly
+noinst_PROGRAMS =  \
+    start_pready      \
+    start_pready_wait \
+    start_pready_range      \
+    start_pready_range_wait \
+    start_pready_list       \
+    start_pready_list_wait  \
+    parrived                \
+    parrived_wait
+
+start_pready_wait_CPPFLAGS = -DTEST_WAIT $(AM_CPPFLAGS)
+start_pready_wait_SOURCES  = start_pready.c
+
+start_pready_range_CPPFLAGS = -DTEST_PREADY_RANGE $(AM_CPPFLAGS)
+start_pready_range_SOURCES  = start_pready.c
+
+start_pready_range_wait_CPPFLAGS = -DTEST_WAIT -DTEST_PREADY_RANGE $(AM_CPPFLAGS)
+start_pready_range_wait_SOURCES  = start_pready.c
+
+start_pready_list_CPPFLAGS = -DTEST_PREADY_LIST $(AM_CPPFLAGS)
+start_pready_list_SOURCES  = start_pready.c
+
+start_pready_list_wait_CPPFLAGS = -DTEST_WAIT -DTEST_PREADY_LIST $(AM_CPPFLAGS)
+start_pready_list_wait_SOURCES  = start_pready.c
+
+parrived_wait_CPPFLAGS = -DTEST_WAIT $(AM_CPPFLAGS)
+parrived_wait_SOURCES  = parrived.c
```

Comment:
I see a terminator $ at the last line when enabling :set line with vim. Could you please share with me your method to detect the issue?

Reply:
I was just looking at github per-commit changes. It shows a red circle if the last line is missing a newline. 

I suspect some of the notes are confusing because I was looking at per-commit changes and they disapper when the next commits combines. If if is too difficult for you to locate the offending commits, you can ignore my notes. Just make sure the final files are all terminated.

Reply:
Added a new line at end